doc/developer-guide/generic_type.md

```diff
@@ -0,0 +1,127 @@
+# Generic Type
+
+## Generic Class
+
+### with type arguments
+``` TypeScript
+class A<T> {
+    _x: T;
+
+    constructor(x: T) {
+        this._x = x;
+    }
+    bar<T>(a: T) {
+        console.log('bar: ' + a);
+    }
+}
+```
+
+#### Creation of class type
```

Comment:
```suggestion
#### Creation of generic class
```

Reply:
I will delete the document first. After the document modification is completed, I will create a new PR and submit it to the internal Repo.